### PR TITLE
Delete external storage permission

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Delete external storage permission  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoMobileApp/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, [accessibility](https://reactnative.dev/docs/accessibility) etc.
- [x] Bump [package](https://github.com/teamsempo/SempoMobileApp/blob/master/package.json#L3) and [android](https://github.com/teamsempo/SempoMobileApp/blame/master/android/app/build.gradle#L141-L142) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoMobileApp/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[App can read/write to External Storage](https://trello.com/c/yinCmsCH/985-app-can-read-write-to-external-storage)

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -6,7 +6,6 @@
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.RECORD_AUDIO"/>
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```
Comment (tristanhcole) [inline android/app/src/main/AndroidManifest.xml:8]: Do we also want to delete READ external storage or is that used by one of our packages?

Comment (michielderoos) [reply android/app/src/main/AndroidManifest.xml]: Good call

**Comment by michielderoos** [comment]
[Dangerous Application Permissions](https://trello.com/c/rhSXlcGT/993-dangerous-application-permissions)

**Review by tristanhcole — APPROVED** [review]
LGTM

**Pull request merged**